app/models/manageiq/providers/vmware/infra_manager/vim_connect_mixin.rb

```diff
@@ -45,13 +45,13 @@ def connection_key(options)
   module ClassMethods
     def raw_connect(options)
       require 'handsoap'
-      require 'VMwareWebService/miq_fault_tolerant_vim'
+      require 'VMwareWebService/MiqVim'
 
-      options[:pass] = ManageIQ::Password.try_decrypt(options[:pass])
-      options[:use_broker] = false
+      ip, user = options.values_at(:ip, :user)
+      pass = ManageIQ::Password.try_decrypt(options[:pass])
```

Comment:
This keeps compatibility with the UI's [`get_task_args`](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/mixins/ems_common/angular.rb#L195)